add rage click issues (#9268)
getsentry/sentry-docs: docs/product/issues/issue-details/replay-issues/index.mdx

---
title: Replay Issues
sidebar_order: 30
description: "Learn about Session Replay rage click issues."
---

> If you've enabled Session Replay, you'll be able to see rage click issues on the **Issues** page in Sentry. Rage clicks, a series of consecutive clicks on the same unresponsive element, are a sure sign of user frustration and most likely deserve your attention.

Reviewer: ```suggestion
If you've enabled [Session Replay](/product/session-replay/), you'll be able to see rage click issues on the **Issues** page in Sentry. Rage clicks, a series of consecutive clicks on the same unresponsive element, are a sure sign of user frustration and most likely deserve your attention.
```

> If you've enabled [Session Replay](/product/session-replay/), you'll be able to see rage click issues on the **Issues** page in Sentry. Rage clicks, a series of consecutive clicks on the same unresponsive element, are a sure sign of user frustration and most likely deserve your attention.

Reviewer 2: ```suggestion
If you've enabled [Session Replay](/product/session-replay/), you'll be able to see rage click issues on the **Issues** page in Sentry. Rage clicks are a series of consecutive clicks on the same unresponsive element. They are a sure sign of user frustration and most likely deserve your attention.
```

> ![Rage click issue](product/issues/issue-details/replay-issues/Rage-click-issue-in-issue-stream.png)

Author: ```suggestion
![Rage click issue](/product/issues/issue-details/replay-issues/Rage-click-issue-in-issue-stream.png)
```

> ![Rage click issue](/product/issues/issue-details/replay-issues/Rage-click-issue-in-issue-stream.png)

Author: Sure!

## Prerequisites for Seeing Rage Clicks

In order to see rage clicks in your issue stream on the **Issues** page, your organization needs to:

- Be sending [session replay events](/product/session-replay/getting-started/)

> - Enable the JavaScript SDK (or related framework), version 7.60.1 or higher

Reviewer: I think this is a little confusing because the [javascript SDK docs](https://docs.sentry.io/platforms/javascript/session-replay/) say version 7.27.0 or higher. I'm also not sure if the same version applies to "related framework".

If the requirements are different for session replays and rage click issues, I think it would help to explicitly call that out.

Author: Great call! @jas-kas could you weigh in?

Reviewer 3: nit: is "(or related framework)" the phrasing?

something like "(or framework specific bundle)" idk

i can confirm that you need sdk v7.60.1 to get rage clicks.
older versions of the sdk will enable session replay, but you will not get rage-click issues from those.

Reviewer 2: ^ +1 what Ryan said. You need to be on a newer SDK version for rage click issues.

> - Enable the JavaScript SDK (or related framework), version 7.60.1 or higher

Author: ```suggestion
- Enable the JavaScript SDK (or framework-specific bundle), version 7.60.1 or higher

<Note>
While you can enable **session replay** with JavaScript SDK version 7.27.0, or higher, you'll need to have version 7.60.1 or higher in order to be able to see **rage click issues**.
</Note>
```

## Get Rage Click Alerts

To set up alerts and get notified when a rage click occurs, follow these steps:

1. Create a new [Alert Rule](https://sentry.sentry.io/alerts/new/issue/) in Sentry.
2. In the "Set conditions" section, set the "IF" filter to "The issue's category is equal to", then choose "Replay" from the dropdown.
3. Add an optional filter if you like.
5. Choose the action to be performed in the "THEN" dropdown.
6. Decide how often you'd like Sentry to look for rage click issues.
7. Lastly, name your alert and add an owner.

## Turn Off Rage Click Issues

> If you don't want to see rage click issues, you can disable them by going to the **Settings** page in Sentry, clicking "Projects" and selecting the project for which you want to disable rage clicks. From here, select "Replays" under the "PROCESSING" heading and toggle off "Create Rage Click Issues".

Reviewer: ```suggestion
If you don't want to see rage click issues, you can disable them by going to the **Settings** page in Sentry, clicking "Projects", and selecting the project for which you want to disable rage clicks. From here, select "Replays" under the "PROCESSING" heading and toggle off "Create Rage Click Issues".
```

> ![Turning off rage click issues](product/issues/issue-details/replay-issues/Turning-off-rage-clicks-toggle.png)

Author: ```suggestion
![Turning off rage click issues](/product/issues/issue-details/replay-issues/Turning-off-rage-clicks-toggle.png)
```

---
title: Rage Click Issues
sidebar_order: 30
description: "Learn about Session Replay rage click issues."
---

> If you've enabled [Session Replay](/product/session-replay/), you'll be able to see rage click issues on the **Issues** page in Sentry. Rage clicks are a series of consecutive clicks on the same unresponsive element. They are a sure sign of user frustration and most likely deserve your attention.

Author: ```suggestion
If you've enabled [Session Replay](/product/session-replay/), you'll be able to see rage click issues on the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page in Sentry. Rage clicks are a series of consecutive clicks on the same unresponsive element. They are a sure sign of user frustration and most likely deserve your attention.
```
